internal/factory/container/container.go

```diff
@@ -552,13 +552,11 @@ func (c *container) AddUnifiedResourcesFromAnnotations(annotationsMap map[string
 		}
 		d, err := b64.StdEncoding.DecodeString(parts[1])
 		// if the value is not specified in base64, then use its raw value.
-		v := ""
 		if err == nil {
-			v = string(d)
+			c.spec.Config.Linux.Resources.Unified[parts[0]] = string(d)
 		} else {
-			v = parts[1]
+			c.spec.Config.Linux.Resources.Unified[parts[0]] = parts[1]
 		}
-		c.spec.Config.Linux.Resources.Unified[parts[0]] = v
```

Comment:
This linter suggestion is an interesting one. The original code does not look all that bad, definitely more readable.

Reply:
That was more or less my decision to get rid of `v`. Do you want to have it back as `var v string`?

Reply:
No worries. Personally, I find the old version slightly easier to read, let it be "easier on the eyes", so to speak. But, we can keep the new one, if that is your preference. :smile: